content/en/meetup/2024-10-30.md

```diff
@@ -4,4 +4,24 @@ location = "TechQuartier, Platz der Einheit 2, floor #3, room name: A66"
 date = 2024-10-30T18:30:00+01:00
 +++
 
-[João Thallis](https://joaothallis.com/) from [BitDevs Porto Alegre](https://poabitdevs.org/) is hosting a BitDevs in Frankfurt. 
+[João Thallis](https://joaothallis.com/) from [BitDevs Porto Alegre](https://poabitdevs.org/) is hosting a BitDevs in Frankfurt.
+
+<details>
+<summary>
+Topics
+</summary>
+
+- Bisq adds Lightning support: Bisq v2.1.0 adds the ability for users to settle trades using the Lightning Network.
+  - https://github.com/bisq-network/bisq2/releases/tag/v2.1.0
+- Samourai wallet fork
+  - https://www.nobsbitcoin.com/ashigaru-v1-0-0/
+- Bitcoin Core project disclosed three vulnerabilities fixed in v25.0
+  - [Public disclosure of three vulnerabilities affecting Bitcoin Core <v25.0](https://groups.google.com/g/bitcoindev/c/WeSDeV8YOSA/m/5zE3n2lxAAAJ)
+  - https://x.com/0xB10C/status/1844705899455062220
+- Mining: Recent problems with testnet4
+  - https://x.com/0xB10C/status/1849133671980339313
+  - [miner: Reorg Testnet4 minimum difficulty blocks](https://github.com/bitcoin/bitcoin/pull/31117)
+- Mutation-core: A mutation testing tool for Bitcoin Core
+  - https://delvingbitcoin.org/t/mutation-core-a-mutation-testing-tool-for-bitcoin-core/1119
+
```

Comment:
Can you add this one: 
"Lightspark unveiled new L2, called Spark"
https://x.com/kphur/status/1850001832787210365

Reply:
And maybe this, even though it's already a month old.   
"Soft-Fork/Covenant Dependent Layer 2 Review from Peter Todd"

https://petertodd.org/2024/covenant-dependent-layer-2-review

Reply:
Will add them!